src/shared/Invoice/InvoicePanel.jsx

```diff
@@ -1,31 +1,46 @@
-import React, { Component } from 'react';
+import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import './InvoicePanel.css';
 
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
+import { isOfficeSite, isDevelopment } from 'shared/constants.js';
 
 import BasicPanel from 'shared/BasicPanel';
 import {
   makeGetUnbilledShipmentLineItems,
   makeTotalFromUnbilledLineItems,
 } from 'shared/Entities/modules/shipmentLineItems';
 
-export class InvoicePanel extends Component {
-  constructor() {
-    super();
-    this.state = {};
-  }
+export class InvoicePanel extends PureComponent {
+  approvePayment = () => {
+    //this.props.onApprovePayment(this.props.shipmentId);
+    console.log('Approve Payment button clicked!');
+  };
 
   render() {
-    let invoicingContent = <span className="empty-content">No line items</span>;
+    let invoicingContent = <div className="empty-content">No line items</div>;
     if (this.props.shipmentLineItems.length > 0) {
       //stand up a table
     }
 
     return (
       <div className="invoice-panel">
-        <BasicPanel title={'Invoicing'}>{invoicingContent}</BasicPanel>
+        <BasicPanel title={'Invoicing'}>
+          {isOfficeSite &&
+            this.props.shipmentState === 'DELIVERED' && (
+              <div className="usa-width-one-whole align-right">
+                <button
+                  className="button button-secondary"
+                  disabled={!this.props.canApprove && !isDevelopment}
```

Comment:
This change is needed for  e2e test to pass
```suggestion
                  disabled={!this.props.canApprove || !isDevelopment}
```